public/pages/CreateMonitor/components/AssociateMonitors/AssociateMonitors.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { Fragment, useState, useEffect, useCallback } from 'react';
+import { EuiSpacer, EuiText } from '@elastic/eui';
+import MonitorsList from './components/MonitorsList';
+import { FormikCodeEditor } from '../../../../components/FormControls';
+import * as _ from 'lodash';
+import { isInvalid, hasError, validateExtractionQuery } from '../../../../utils/validate';
+
+const AssociateMonitors = ({
+  monitors,
+  options,
+  searchType = 'graph',
+  isDarkMode,
+  monitorValues,
+}) => {
+  const [graphUi, setGraphUi] = useState(searchType === 'graph');
+
+  const queryTemplate = {
+    sequence: {
+      delegates: [],
+    },
+  };
+
+  const delegatesToMonitors = (value) =>
+    value.sequence.delegates.map((monitor) => ({
+      label: '',
+      value: monitor.monitor_id,
+    }));
+
+  useEffect(() => {
+    if (monitors?.length) {
+      const value = { ...queryTemplate };
+      monitors.map((monitor, idx) => {
```

Comment:
use `forEach` instead of `map` since we are not using the returned array?

Reply:
Makes sense, done.